src/components/InvolvedCard.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { StaticImage } from "gatsby-plugin-image";
+import "./GetInvolved.css";
+import Card from "react-bootstrap/Card";
+
+export default function InvolvedCard({ imgPath, children, title }) {
+  // Card: https://react-bootstrap.github.io/components/cards/
+  // Don't worry about the close button for now
+  // Use h2 for the title
+  console.log(imgPath);
+  return (
+    <>
+      <Card style={{ width: "18rem" }}>
+        <StaticImage
```

Comment:
Why are there two `StaticImage`s inside the card, and why does this one have a hardcoded image path?